modules/jwplayerBidAdapter.js

```diff
@@ -50,4 +68,76 @@ export const spec = {
   // onBidderError: function({ error, bidderRequest }) {}
 };
 
+function buildRequest(bidRequest, bidderRequest) {
+  const {params} = bidRequest;
+
+  const videoAdUnit = deepAccess(bidRequest, 'mediaTypes.video', {});
+  const videoBidderParams = deepAccess(bidRequest, 'params.video', {});
+
+  const videoParams = {
+    ...videoAdUnit,
+    ...videoBidderParams
+  }
+
+  const video = {
+    w: parseInt(videoParams.playerSize[0][0], 10),
+    h: parseInt(videoParams.playerSize[0][1], 10)
+  }
+
+  // Bid FLoor
+  const bidFloorRequest = {
+    currency: bidRequest.params.cur || 'USD',
+    mediaType: 'video',
+    size: '*'
+  };
+
+  let floorData = bidRquest.params;
```

Comment:
i assume you meant to leave `floorData` undefined ? `bidRequest.params` wouldn't make sense for it probably